### PR TITLE
pdfpc 4.6.0

### PR DESCRIPTION
**Pull request by chenrui333**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

continuing #118911

### CODE REVIEW

**Comment by chenrui333** [comment]
```
--   No package 'webkit2gtk-4.0' found
  CMake Error at /home/linuxbrew/.linuxbrew/Cellar/cmake/3.27.8/share/cmake/Modules/FindPkgConfig.cmake:607 (message):
    A required package was not found
  Call Stack (most recent call first):
    /home/linuxbrew/.linuxbrew/Cellar/cmake/3.27.8/share/cmake/Modules/FindPkgConfig.cmake:[82](https://github.com/Homebrew/homebrew-core/actions/runs/6986179004/job/19011401575?pr=155471#step:4:83)9 (_pkg_check_modules_internal)
    src/CMakeLists.txt:33 (pkg_check_modules)
```

File: Formula/p/pdfpc.rb [hunk 1]
```diff
@@ -18,34 +18,33 @@ class Pdfpc < Formula
...
   depends_on "gtk+3"
+  depends_on "json-glib"
   depends_on "libgee"
   depends_on "librsvg"
+  depends_on "libsoup"
   depends_on "poppler"
+  depends_on "qrencode"
+
+  on_linux do
+    depends_on "webkitgtk"
```
Comment (chenrui333) [inline Formula/p/pdfpc.rb:35]:
```
-- Checking for module 'webkit2gtk-4.0'
  --   No package 'webkit2gtk-4.0' found
  CMake Error at /home/linuxbrew/.linuxbrew/Cellar/cmake/3.27.8/share/cmake/Modules/FindPkgConfig.cmake:607 (message):
    A required package was not found
  Call Stack (most recent call first):
    /home/linuxbrew/.linuxbrew/Cellar/cmake/3.27.8/share/cmake/Modules/FindPkgConfig.cmake:[82](https://github.com/Homebrew/homebrew-core/actions/runs/6986179004/job/19011401575?pr=155471#step:4:83)9 (_pkg_check_modules_internal)
    src/CMakeLists.txt:33 (pkg_check_modules)
```

Comment (iMichka) [reply Formula/p/pdfpc.rb:35]: Interestingly, we install `include/webkitgtk-4.1/` / `libexec/webkit2gtk-4.1/`, not version `4.0` ..

`pdfpc` is expliciltely looking for `4.0` here: https://github.com/pdfpc/pdfpc/blob/14344bc203cfff71a1b72deb161e003c9b480bb2/src/CMakeLists.txt#L35-L38

Comment (chenrui333) [reply Formula/p/pdfpc.rb:35]: Good find, let me raise it up.

**Comment by chenrui333** [comment]
actually it relates to:
- https://github.com/pdfpc/pdfpc/issues/664
- https://github.com/pdfpc/pdfpc/issues/671

**Comment by chenrui333** [comment]
Gonna close the PR for now (due to the upstream issues)

**Pull request closed without merging**